chalicelib/responseobjects/hca_response_v5.py

```diff
@@ -124,26 +127,49 @@ class ManifestResponse(AbstractResponse):
     """
     Class for the Manifest response. Based on the AbstractionResponse class
     """
+
+    def _translate(self, untranslated, keyname):
+        m = self.manifest_entries[keyname]
+        return [untranslated[es_name] if es_name in untranslated else "" for es_name in m.values()]
+
     def return_response(self):
-        return self.apiResponse
+        es_search = self.es_search
+
+        headers = {'Content-Disposition': 'attachment; filename="export.tsv"',
+                   'Content-Type': 'text/tab-separated-values'}
+
+        output = StringIO()
+
+        writer = csv.writer(output, dialect='excel-tab')
 
-    def __init__(self, raw_response, manifest_entries, mapping):
+        writer.writerow(list(self.manifest_entries['bundles'].keys()) + list(self.manifest_entries['files'].keys()))
```

Comment:
I don't think it is safe to assume that every iteration over `self.manifest_entries['bundles']` and `self.manifest_entries['files']` yields the entries in the same order. Can you please make those to mappings `OrderedDict`s?